package.json

```diff
@@ -2,54 +2,60 @@
   "name": "ember-responds-to",
   "version": "1.4.0",
   "description": "Ember mixins for browser event handling.",
+  "keywords": [
+    "ember-addon",
+    "responds to",
+    "events",
+    "keydown",
+    "resize",
+    "scroll"
+  ],
+  "license": "MIT",
+  "author": "",
+  "contributors": [
+    {
+      "name": "Robert Wagner",
+      "email": "rwwagner90@gmail.com",
+      "url": "https://github.com/rwwagner90"
+    }
+  ],
   "directories": {
     "doc": "doc",
     "test": "tests"
   },
+  "repository": "git://github.com/dollarshaveclub/ember-responds-to.git",
   "scripts": {
     "start": "ember server",
     "build": "ember build",
-    "lint": "eslint .",
-    "test": "ember try:testall"
+    "test": "ember try:each"
   },
-  "repository": "git://github.com/dollarshaveclub/ember-responds-to.git",
-  "engines": {
-    "node": ">= 0.10.0"
+  "dependencies": {
+    "ember-cli-babel": "^6.8.1"
   },
-  "author": "",
-  "license": "MIT",
   "devDependencies": {
-    "broccoli-asset-rev": "^2.0.2",
-    "ember-cli": "2.12.1",
+    "broccoli-asset-rev": "^2.4.5",
+    "ember-cli": "~2.15.0-beta.2",
```

Comment:
It really doesn't make a difference for addons. Only things in dependencies have any effect on the consuming application. devDependencies are just for developing the addon.